### PR TITLE
2 fixes

### PR DESCRIPTION
**Pull request by seanm**

### CODE REVIEW

**Comment by jcfr** [comment]
Thanks @seanm 

Appveyor error is a false alarm (@thewtex: Should we disable the INSIDEITK case?)

File: src/metaUtils.cxx [hunk 1]
```diff
@@ -28,7 +28,7 @@ inline bool IsBlank(int c)
 
 #include "metaUtils.h"
 
-//#include <cassert>
+#include <cassert>
```
Comment (jcfr) [inline src/metaUtils.cxx]: Should this header be removed instead ?

**Comment by jcfr** [comment]
Merging. As already mention in https://github.com/Kitware/MetaIO/pull/37#pullrequestreview-59636930, failure are false positive.

**Comment by seanm** [comment]
Sweet, thanks.

Are you accustomed to syncing up VTK & ITK?

**Comment by jcfr** [comment]
> Are you accustomed to syncing up VTK & ITK?

Yes.

It is now straighforward. In either VTK or ITK, create a topic like  `update-metaio`, and then for run this script:

* for ITK: https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/ThirdParty/MetaIO/UpdateFromUpstream.sh
* for VTK: https://gitlab.kitware.com/vtk/vtk/blob/master/Utilities/MetaIO/update.sh

Et voila,

**Comment by thewtex** [comment]
Yes, we can disable the `ISINSIDEITK` checks -- they are covered by other tests.

**Pull request merged**